styles/style-officers.css

```diff
@@ -92,7 +93,7 @@ main {
 }
 
 .page-title {
-    background-color: var(--kojobs-gray); 
+    background-color: var(--kojobs-green); 
```

Comment:
don't need to change background color for officers page, only for events page :)